Source/Icebreaker/Cards/UnrecognizedInputCard.cs

```diff
@@ -23,11 +23,12 @@ public class UnrecognizedInputCard
         public static Attachment GetCard()
         {
             var baseDomain = CloudConfigurationManager.GetSetting("AppBaseDomain");
-            var htmlUrl = Uri.EscapeDataString($"https://{baseDomain}/Content/tour.html?theme={{theme}}");
             var tourTitle = Resources.WelcomeTourTitle;
             var appId = CloudConfigurationManager.GetSetting("ManifestAppId");
-            var tourUrl = $"https://teams.microsoft.com/l/task/{appId}?url={htmlUrl}&height=533px&width=600px&title={tourTitle}";
-            var tourButtonText = Resources.TakeATourButtonText;
+
+            var htmlUrl = Uri.EscapeDataString($"https://{baseDomain}/Content/tour.html?theme={{theme}}");
+            var escapedTourTitle = Uri.EscapeDataString(tourTitle);
+            var escapedTourUrl = Uri.EscapeDataString($"https://teams.microsoft.com/l/task/{appId}?url={htmlUrl}&height=533px&width=600px&title={escapedTourTitle}");
```

Comment:
@coderkrishna Don't escape the URL itself. We talked about this specific point.

Reply:
Understood, my mistake @aosolis